docs/index.rst

```diff
@@ -273,7 +275,8 @@ The 3.0.0 release of Crane adds support for the Docker v2 API manifest lists sch
 Crane now supports version 4 of the json data file produced by Pulp. The new version enables
 Crane to serve manifest lists. In order to decide to which manifest type to redirect
 the request, Crane looks for an optional accept header specified in the request and based on this
-information redirects whether to the manifest list or image manifest.
+information redirects whether to the manifest list or image manifest. The search will also return results
```

Comment:
This should under 3.1.0 version

Please create a new paragraph where you'd mention the v2 docker search.